### DISCUSSION TITLE
the total length of selected lines

### DISCUSSION THREAD

**Discussion by faon-92** [SubtitleEdit/subtitleedit#7408] [Q&A]
Hi everyone,
Is there any way to know the total length of the selected lines? I know that when two lines are selected, the total length is displayed next to the profile in the bottom-right corner. However, when more than two lines are selected, I don't see any such indication.

many thanks

**Comment by niksedk** [comment]
In latest beta it's possible by right-clicking on selected lines in the list view, and choose "Statistics":
![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/2dc4e2b6-0e90-4169-a3b0-94e9687de412)

Latest beta link: https://github.com/SubtitleEdit/subtitleedit/releases/download/4.0.1/SubtitleEditBeta.zip
